### PR TITLE
remove drawio submodule, install from npm

### LINKED ISSUE #27: Replace drawio submodule with published packages
The `npm` packages for `@deathbeds/jupyterlab-drawio` are up:

## the good

```
jupyter labextension install \
  @deathbeds/jupyterlab-drawio-webpack@13.6.1-alpha0 \
  @deathbeds/jupyterlab-drawio@0.8.0-alpha0
```

Installing these directly would replace the use of the enormous, nested submodules, and make things snappier and more predictable, but crucially less complex.

## the bad

The pdf is still not really shippable (random npm/browser installs), and would require a `pip` dependency, so may not be worth it for now. I've also found that some complex shapes (e.g. the cloud icons) don't embed properly.

```
conda install -yc conda-forge requests_cache pypdf2
pip install jupyter-drawio-export@0.8.0.a0
jupyter labextension install @deathbeds/jupyterlab-drawio-pdf@0.8.0-alpha0
```

## the ugly

Notebooks work,  but are separated out:
```bash
jupyter labextension install @deathbeds/jupyterlab-drawio-notebook@0.8.0-alpha0
````
This is mostly because I have a lot of stuff planned there. Also, the notebook metadata tag changed for `.dio.ipynb` to add the `@deathbeds` namespace, but is otherwise compatible.

### PR DESCRIPTION
**Pull request by nrbgt**
[Binder](https://mybinder.org/v2/gh/nrbgt/ipyradiant/gh-27-drawio-from-npm?urlpath=lab%2Ftree%2Fdesign%2Farchitecture%2Farchitecture.dio.svg)

- fixes #27
- [x] remove submodule
  - [x] remove from dodo, preflight, etc.
- [x] add labextensions
- [x] upgrade lab
- [x] resolve envs

This does not add the PDF stuff, as it's not fully-baked yet

### CODE REVIEW

**Comment by nrbgt** [comment]
Still not sure if splitting the commands up will give us better output... having some really slow network, so it's not showing the logs interactively...

**Comment by nrbgt** [comment]
have rolled back, this is ready to go. it'll be worth (ancedotally) comparing the runtimes, but I'm pretty confident it will at least be a few minutes shorter.

File: .github/workflows/ciV0.yml [hunk 1]
```diff
@@ -66,8 +64,7 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          # presently fails during some node-based stuff
-          # - macos-latest
```
Comment (nrbgt) [inline .github/workflows/ciV0.yml:70]: :tada:

File: examples/Tab_App_Example.ipynb [hunk 2]
```diff
@@ -1,135 +1,135 @@
...
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.7.8"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 4
-}
\ No newline at end of file
+{
```
Comment (nrbgt) [inline examples/Tab_App_Example.ipynb:1]: presumably this is line endings. remember, ONLY YOU can:

```bash
git config --set core.autocrlf false
```

File: postBuild [hunk 3]
```diff
@@ -9,16 +9,6 @@ python -m pip install -e . --no-cache-dir --no-deps
 
 # lab
 jupyter labextension install $(cat labextensions.txt) --no-build --debug
-
-pushd vendor/jupyterlab-drawio
-    git submodule update --init
-    jlpm --ignore-optional
-    jlpm build
```
Comment (nrbgt) [inline postBuild:16]: since it's not doing tsc anymore, osx should be fine

**Comment by nrbgt** [comment]
Upgraded to `alpha1`, just for giggles (not really any new features, just better tested).

File: examples/Tab_App_Example.ipynb (hunk 2 above)
Comment (zwelz3) [reply examples/Tab_App_Example.ipynb:1]: I've run this in all my git repositories. Is there a clean way to add to global git config?

Comment (nrbgt) [reply examples/Tab_App_Example.ipynb:1]: that's the thing, right there.

**Review by zwelz3 — APPROVED** [review]
Everything looks good and build locally. 💪 
Anecdotal, but it is launching the drawio lab waaaay faster on my Windows machine.

**Pull request merged**